static/js/init.js

```diff
@@ -114,6 +115,9 @@ $(document).ready(function() {
     new Search($(this));
   });
 
+  // Initialize table of contents
+  new toc.TOC('.js-toc');
```

Comment:
Didn't notice this before, but this looks different from our convention of passing in a set of jquery elements, rather than a selector string. I'm fine with either approach, but would prefer to keep it consistent.